### PR TITLE
Hackdna/update apache config

### PR DESCRIPTION
**Pull request by hackdna**
Lower HTTP timeout value and remove unnecessary values from Apache log

### CODE REVIEW

File: deployment/refinery-modules/refinery/manifests/apache2.pp [hunk 1]
```diff
@@ -1,9 +1,9 @@
 class refinery::apache2 {
 
-  # must be set to the ELB Idle Timeout value provided in the CloudFormation template
-  # for chunked uploads of large files (MD5 calculation on the server can take a long time)
-  # to be adjusted to a more reasonable value after enabling file uploads to S3
-  $timeout = 1800
+  # must be set to the ELB IdleTimeout value in the CloudFormation template
+  # this is higher than the default of 60 to accommodate long synchronous
+  # processing of metadata during data set import
+  $timeout = 180  # seconds
```
Comment (mccalluc) [inline deployment/refinery-modules/refinery/manifests/apache2.pp]: How did you pick this number? If the time increases with larger files, how big a file will typically work in 180s?

(If you're tweaking the comment, could you capitalize the first letter and add a period at the end?)

File: deployment/refinery-modules/refinery/manifests/apache2.pp [hunk 2]
```diff
@@ -40,6 +40,11 @@
     content => "AcceptFilter http none\nAcceptFilter https none",
   }
 
+  # log Django messages only
+  apache::custom_config { 'error-log-format':
+    content => 'ErrorLogFormat "%M"',
+  }
+
```
Comment (mccalluc) [inline deployment/refinery-modules/refinery/manifests/apache2.pp]: *Only* the error messages? Time and perhaps some request info could be useful...

File: deployment/refinery-modules/refinery/manifests/apache2.pp (hunk 1 above)
Comment (hackdna) [reply deployment/refinery-modules/refinery/manifests/apache2.pp]: 180 seconds was more than enough to import each of the several randomly selected ISA-archives. In any case, HTTP requests should not take this long, so if there is more time required to process metadata then this is a larger problem that should be addressed in the metadata processing code.

[Puppet style guide](https://puppet.com/docs/puppet/5.4/style_guide.html#comments) does not address capitalization or punctuation, so I think this is acceptable.

File: deployment/refinery-modules/refinery/manifests/apache2.pp (hunk 2 above)
Comment (hackdna) [reply deployment/refinery-modules/refinery/manifests/apache2.pp]: Django error messages contain timestamps. Request info goes into the access log.

File: deployment/refinery-modules/refinery/manifests/apache2.pp (hunk 1 above)
Comment (scottx611x) [reply deployment/refinery-modules/refinery/manifests/apache2.pp]: I thought we bumped this value initially to handle large file uploads?

Comment (hackdna) [reply deployment/refinery-modules/refinery/manifests/apache2.pp]: Large file uploads are sent directly to S3.

**Pull request merged**